### DISCUSSION TITLE
Ukrainian Translation (completed)

### DISCUSSION THREAD

**Discussion by bitrate-pgpk** [aclap-dev/video-downloadhelper#1404] [Language & Translation]
[uk.txt](https://github.com/user-attachments/files/17528447/uk.txt)

**Comment by Bartok22** [comment]
Hello!  
I have translated the last 14 lines. I am attaching the file for your review!
[uk.txt](https://github.com/user-attachments/files/18550254/uk.txt)

**Comment by ustav** [comment]
Here is an updated file with added strings
[uk.txt](https://github.com/user-attachments/files/19840193/uk.txt)
